docs/database-functionalities/triggers.md

```diff
@@ -0,0 +1,295 @@
+---
+id: triggers
+title: Triggers
+sidebar_label: Triggers
+---
+
+This article is a part of a series intended to showcase Memgraph's features
+and bring the user up to speed on developing with Memgraph.
+
+We highly recommend checking out the other articles from this series which
+are listed in our [database functionalities section](./database-functionalities.md).
+
+## Introduction
+
+Memgraph supports running openCypher statements after a certain event happens during transaction execution, i.e. triggers.
+
+As user, you can create triggers, delete them, and print them out.
+All the triggers are persisted on the disk, so no information is lost between the runs.
+
+## Creating a trigger
+
+To create a new trigger, a query of the following format should be used:
+
+```plaintext
+CREATE TRIGGER trigger_name ( ON ( () | --> ) CREATE | UPDATE | DELETE )
+( BEFORE | AFTER ) COMMIT
+EXECUTE openCypherStatements
+```
+
+At first, the query may seem complex, so let's talk about the different parts of it independently.
+
+### Trigger name
+Each created trigger should be defined with a name. The name needs to be unique globally.
+E.g. two triggers for different event still can't have the same name.
+
+### Event type
+Optionally, users can define for which event should the trigger execute its statements.
+The event type is defined using the following part:
+
+```plaintext
+ON ( () | --> ) CREATE | UPDATE | DELETE
+```
+
+There are three main event types:
+  - CREATE
+  - UPDATE
+  - DELETE
+
+For each of the event types, user can denote if he wants to execute trigger's statements just on the events
```

Comment:
There are mixed usage of singular and plural form of user, so I would change everything to plural form, because that's more general in my opinion.
Sticking to singular form is also fine, then we are missing a few `a`/`the`.